bypass_url_parser.py

```diff
@@ -262,13 +262,22 @@ def _generate_curls(self, url_obj: ParseResult):
 
         # [http_headers_scheme] - Custom scheme rewrite with X-Forwarded-Scheme
         if any(mode in ["all", "http_headers_scheme"] for mode in self.current_bypass_modes):
-            for const_header_scheme in self.const_header_schemes:
-                for const_proto in self.const_protos:
-                    cmd = [*self.base_curl, "-H", f"{const_header_scheme}: {const_proto}", target_url]
-                    item = CurlItem(url_obj, self.base_curl, cmd, bypass_mode="http_headers_scheme",
-                                    target_ip=url_public_ip, debug=self.debug, ext_logger=self.logger)
-                    if item not in self.curl_items:
-                        self.curl_items.append(item)
+            commands = set()
+            for const_proto in self.const_protos:
+                # Specific rule for header 'Forwarded: proto='
+                commands.add(tuple([*self.base_curl, "-H", f"Forwarded: proto={const_proto}", target_url]))
+                for const_header_scheme in self.const_header_schemes:
+                    # Standard headers ending with "-Proto" or "-Scheme"
+                    commands.add(tuple([*self.base_curl, "-H", f"{const_header_scheme}: {const_proto}", target_url]))
+            # Adding non-standard headers that take 'on' value (Ex: Microsoft)
+            for on_header in ["Front-End-Https", "X-Forwarded-HTTPS", "X-Forwarded-SSL"]:
```

Comment:
Should we move them to a flat file like other headers ?

Reply:
It's possible but I think it's a bit overkill to create a new file for 3 very specific headers that will never change and won't take other values than `on`.

For more readability and to avoid having too many hardcoded headers scattered in the code, we can also put them in a constant (ex: `Bypasser.HEADERS_SCHEME_OTHER = ["Front-End-Https", "X-Forwarded-HTTPS", "X-Forwarded-SSL"]`)

Reply:
Hmmm... Might be, but as we already the utils to load flat files, I'd rather keep it that way or move it to an external file ! :)
Having them in a variable would help for further processing, and similar behaviors, but fow now that's ok I guess

Reply:
Ok, so for now, no file nor constant. No need for these non-standard scheme headers elsewhere in the code.